shared/fs/row/rows.js

```diff
@@ -10,59 +10,80 @@ import Tlf from './tlf-container'
 import Still from './still-container'
 import Editing from './editing-container'
 import Uploading from './uploading-container'
+import {rowHeight} from './common'
 
 type Props = {
   items: Array<Types.RowItem>,
   routePath: I.List<string>,
+  inDestinationPicker?: boolean,
   ifEmpty?: ?React.Node,
 }
 
-export const WrapRow = ({children}: {children: React.Node}) => (
+export const WrapRow = ({children, noDivider}: {children: React.Node, noDivider?: boolean}) => (
```

Comment:
Can you make the divider a positive case instead of a negative case? i.e. `showDivider`

I know this means you'll have to plumb it through the existing code, but at least it'll be easier to read overall.

Reply:
Good point. Although I'm realizing we seem to always want the divider. So I'll probably just revert this part.